.github/workflows/helm-release.yaml

```diff
@@ -3,7 +3,10 @@ name: Release Helm Chart
 on:
   push:
     tags:
-      - "*\\.*\\.*"  # only run on version tags
+      # 2023.9.1, but not 2023.09.01
+      # 2023.11.5, but not 2023.11.05
+      # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
+      - '2[0-9][2-9][3-9].1?[0-9].[1-3]?[0-9]'
```

Comment:
lol king of regex 👑 

![image](https://github.com/PrefectHQ/prefect-helm/assets/52710/fe32ca9f-6821-4ff3-aa64-e835c8a731cc)

Reply:
i made it so that we're good until the year 2999